protonmail/attachments.go

```diff
@@ -33,6 +34,9 @@ type Attachment struct {
 	Signature string
 
 	unencryptedKey *packet.EncryptedKey
+
+	// FORGIVE ME: I could not figure out how to use string signature
+	SignatureBuff  bytes.Buffer `json:"-"`
```

Comment:
`bytes.Buffer.String` should return the string. You can populate `Signature` with it instead of adding this new field.